github/users_gpg_keys.go

```diff
@@ -40,6 +40,28 @@ type GPGEmail struct {
 	Verified *bool   `json:"verified,omitempty"`
 }
 
+// GetGPGKeys fetches GPG keys for a user by name.
+//
+//GitHub API docs: https://developer.github.com/v3/users/gpg_keys/#list-gpg-keys-for-a-user
```

Comment:
Minor, don't forget a space between the slashes and the comment text.

Rationale: https://dmitri.shuralyov.com/idiomatic-go#comments-for-humans-always-have-a-single-space-after-the-slashes.